src/triggerReader_cross.cc

```diff
@@ -962,10 +969,14 @@ bool triggerReader_cross::isVBFfired  (Long64_t triggerbit_1, Long64_t matchFlag
   bool ptCut = false;
   bool etaCut1 = false;
   bool etaCut2 = false;
+
+  int temp = 0;
+  if (!pass_triggerbit) pass_triggerbit = &(temp);
```

Comment:
pass_triggerbit is initialized to null in triggerReader_cross.h; here it's just set to the correct type *int in case no pass_triggerbit is passed to isVBFfired